docs/docs/policies.mdx

```diff
@@ -732,33 +707,32 @@ default restart, label `2` a greeting and `3` indicates goodbye.
 
 It's often useful to include a bit more history than just the current state
 when predicting an action. The `TrackerFeaturizer` iterates over tracker
-states and calls a `SingleStateFeaturizer` for each state. There are two
-different tracker featurizers:
+states and calls a `SingleStateFeaturizer` for each state to create numeric
+input features for a policy.
+The target labels correspond to bot actions or bot utterances
+represented as index of a corresponding action in the domain.
```

Comment:
Is this needed? Might be confusing, not sure.

`a corresponding action` -> `the corresponding action`

Reply:
the whole phrase is confusing?
what about
```
The target labels correspond to bot actions or bot utterances
represented as index in a list of all possible actions.
```